conf/messages.cy

```diff
@@ -213,13 +225,19 @@ consigneeExport.checkYourAnswers.label = Goods being exported outside of the UK
 consigneeExport.error.required = Select yes if the goods are being exported outside of the UK and EU
 consigneeExport.change.hidden = are the goods being exported outside of the UK and EU
 
-checkYourAnswersJourneyType.title = Check your answers
-checkYourAnswersJourneyType.heading = Check your answers
+consigneeExportVat.title = Do you know the VAT or EORI number of the person representing the consignor at the office of export?
+consigneeExportVat.heading = Do you know the VAT or EORI number of the person representing the consignor at the office of export?
+consigneeExportVat.yesVatNumber = Yes - VAT number
+consigneeExportVat.yesEoriNumber = Yes - EORI number
+consigneeExportVat.no = No
+consigneeExportVat.checkYourAnswers.label = VAT or EORI number of the person representing the consignor
+consigneeExportVat.error.required = Select consigneeExportVat
+consigneeExportVat.change.hidden = VAT or EORI number of the person representing the consignor
+consigneeExportVat.vatNumber.label = VAT number
+consigneeExportVat.eoriNumber.label = EORI number
+consigneeExportVat.consigneeExportType.error.required = Select 'no' if you don't know the VAT or EORI number
```

Comment:
the apostrophes are not showing on the page when compiled.

Reply:
https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/38/commits/ee10b3daa3c5a2b53c40c937f6ebd6493c4d3b2f fixes this